Datathon ops page design

AlvinCao88 opened the pull request:
# Description

<!-- Please include a summary of the change -->
Adjusted and fix responsiveness and design of the page 

Relevant GitHub Issues that will be fixed are
Reconstruct the operations site QR page responsive #392
## Type of change

<!-- Select all options that are relevant. -->

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update
- [x] Chore (anything else)

## How Has This Been Tested?

<!-- Summary of tests conducted -->

- [x] Responsiveness on various devices
- Desktop Browser
  - [ ] Apple Safari
  - [x] Google Chrome
  - [ ] Microsoft Edge
  - [ ] Mozilla FireFox
- Mobile Browser
  - [ ] Apple Safari
  - [ ] Google Chrome
  - [ ] Microsoft Edge
  - [ ] Mozilla FireFox
- Other Tests:

## Checklist

- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation

## PR Closure

- Use **Squash And Merge** to Close Pull Request
- Delete any branch after merging

ThunderRoar committed:
fix: 🚑️ update packages

AlvinCao88 committed:
style: 🎨 Adjusted the style and responsiveness of the volunteer scan page

added a header for the nav bar, removed the scrolling of the qrcode, increase the size of the qrcode, fix the style of the qrcode, increased the time and adjusted the style of the participant page

ThunderRoar reviewed (changes requested):
1. In light mode, the sign in page for volunteers camouflages with the login component background making text invisible.

2. Testing on mobile, the logout button becomes larger which disrupts the navigation bar as well. Try to fix this if its a quick thing else no issues.

3. For the "Show Camera" button, have the icon and text appear in the same line.

4. After scanning a QR code, the user is unable to scroll to see the available services and buttons.

Other than those, everything else looks great!

AlvinCao88 committed:
style: 🎨 Feedback from PR

Remove the white background, adjusted the camera icon with the name and added the scroll back to the info page

ThunderRoar reviewed (approved):
Looks good
